resources/views/external-link-confirm.blade.php

```diff
@@ -6,61 +6,97 @@
 
 @php($identifier = md5($url . '-' . Str::random(8)))
 
-<a
-    href="javascript:;"
-    x-data="{
+<div x-data="{
         openModal() {
             Livewire.emit('openModal', '{{ $identifier }}')
+        },
+        redirect() {
+            window.open('{{ $url }}', '_blank')
+        },
+        hasDisabledLinkWarning() {
+            return localStorage.getItem('has_disabled_link_warning') === 'true';
         }
     }"
     class="inline-block items-center space-x-2 font-semibold break-all cursor-pointer link"
-    @click="openModal"
 >
-    <span>{{ $text ?? $slot ?? '' }}</span>
-
+    <a
+        :href="hasDisabledLinkWarning() ? '{{ $url }}' : 'javascript:;'"
+        :target="hasDisabledLinkWarning() ? '_blank' : '_self'"
+        rel="noopener nofollow"
+        class="inline-flex items-center space-x-2 font-semibold whitespace-nowrap cursor-pointer link"
+        @click="hasDisabledLinkWarning() ? redirect() : openModal()"
+    >
+        <span>{{ $text ?? $slot ?? '' }}</span>
+    </a>
     @unless($noIcon)
```

Comment:
it used to be possible to click on the icon, that's no longer a thing after these changes. So if you click the icon, it doesn't open the external link modal